filters/scheduler/lifo.go

```diff
@@ -0,0 +1,335 @@
+package scheduler
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/aryszka/jobstack"
+	log "github.com/sirupsen/logrus"
+	"github.com/zalando/skipper/filters"
+	"github.com/zalando/skipper/scheduler"
+)
+
+// TODO: must be documented that it cannot be used together with the legacy shunting, meaning
+// that it's incompatible with MarkServed().
```

Comment:
@aryszka I added a comment on the `redirect()` filter, is this good enough or do you have any other recommendations before merging this PR?

Reply:
looks good

Reply:
we can drop this todo